lib/Resque/Log.php

```diff
@@ -10,8 +10,11 @@ class Resque_Log extends Psr\Log\AbstractLogger
 {
 	public $verbose;
 
-	public function __construct($verbose = false) {
+    public $debug;
```

Comment:
@danhunsaker i can understand that, was a temporary fix in my personal fork, maybe if is possible we can discuss for make a correct refresh of the project with namespace and more stuff